Add news post for release 20150424

obrakmann opened the pull request:
Eye candy is still missing

pchote commented on content/news/release-20150412.md:
```diff
@@ -0,0 +1,41 @@
+We are very pleased to announce the Spring release of OpenRA.  This is a huge step forward for the Dune 2000 mod in particular, but also, behind the scenes, for the Tiberian Sun mod which has received support for some crucial features during this cycle.
```
"... for our (work in progress, and still unreleased) Tiberian Sun mod ..."

pchote commented on content/news/release-20150412.md:
```diff
@@ -0,0 +1,41 @@
+
+This release includes a total of over 1400 changes by 53 contributors, the culmination of five and a half months of relentless work, and with almost 2000 files and more than 210,000 changed lines one of the busiest development cycles to date.
+
+The development focus has been on the Dune 2000 mod, which has made some huge steps forward and contains the most visible changes. A new sidebar, sandworms and carryalls are the big new features, which should make any Dune 2000 player feel right at home in our mod. The other mods were not neglected, though. In Red Alert, the standout feature is the support of playable factions such as Russia, Eng ...
```
"A large development focus" might be a bit more correct.

pchote replied:
and probably "more at home" to try and avoid complaints about our balance situation.

pchote replied:
"both campaign and stand-alone ones" is a little redundant, and removing it may make the sentence flow a bit better.

penev92 replied:
"A large development focus" sounds wrong to me :|

pchote commented on content/news/release-20150412.md:
```diff
@@ -0,0 +1,41 @@
+
+Of course, we have not only improved the mods themselves. Here is a list of changes that affect the game in general:
+
+* The server browser now shows games across all mods, allowing to join a game from a different mod than the one currently loaded.
```
It would be nice if we could get this down to one line.

pchote commented on content/news/release-20150412.md:
```diff
@@ -0,0 +1,41 @@
+   * Friendly units blocking the exit of a production building will now move out of the way if needed.
+   * Units will no longer hunt cloaked units.
+   * The spy plane in Red Alert will no longer lure your anti-air capable units across the map.
+   * Units that have been loaded into a transport will now be correctly removed from the minimap.
```
I think there's a couple to many of these minor points - this one is probably the least important, and could be removed.

pchote commented on content/news/release-20150412.md:
```diff
@@ -0,0 +1,41 @@
+
+Looking forward, we hope to get the next release out faster than this time around (famous last words!). Feature-wise, you can probably expect the first appearance of the in-game map editor in the next release. Beyond that, we will see... If you want to keep track of the development, keep an eye on the [release milestone](https://github.com/OpenRA/OpenRA/milestones/Next%20release) and the [develop ...
+
+We will also try to make more use of [our YouTube channel](https://www.youtube.com/channel/UCRoiPL1J4K1-EhQeNazrYig) to show off new and little known existing features, provide mission walkthroughs and so on, so watch and subscribe!
```
Maybe add a sentence saying that we will also include the best player-submitted videos on our channel? (we really should start doing this)

obrakmann commented:
Updated, now with a few images

pchote commented on content/news/release-20150412.md:
```diff
@@ -0,0 +1,59 @@
+
+The long awaited sandworms and Carryalls have finally arrived in our Dune 2000 mod!
+
+![Playable countries](/images/news/20150412-ra-country-dropdown.png)
```
Another missing mouse cursor :)

pchote commented on content/news/release-20150412.md:
```diff
@@ -0,0 +1,59 @@
+
+Some missions have received support for multiple difficulty levels.
+
+![Refund tooltip](/images/news/20150412-d2k-refund-tooltip.png)
```
Here too.

penev92 commented on content/gamenews:
```diff
@@ -1,4 +1,4 @@
+	Title: New OpenRA release out now! This is not a drill!
 	Author: Oliver Brakmann
-	Content: The Red Alert balancing adjustments have been finished and thus warrant another playtest, which will hopefully be the last before our next long-awaited release.\n\nBesides the RA balancing, which represents the bulk of the changes, some minor fixes and polishing are included. For the detailed list, please see our changelog at http://changelog.openra.net as usual.\n\nThis will likely be  ...
+	Content: We are thrilled to announce the Spring release of OpenRA!\n\nThis release, five and a half months in the making, is one of our biggest yet, with countless numbers of new features, bug fixes and improvements.  Sandworms and Carryalls for Dune 2000 and playable Countries in Red Alert are the most visible enhancements, but there is lots more to discover!\n\nCheck out http://openra.net for  ...
```
- `with countless numbers of new features` -> `with countless new features`
 - you have 2 spaces before `Sandworms and Carryalls`

penev92 commented on content/news/release-20150412.md:
```diff
@@ -0,0 +1,59 @@
+We are very pleased to announce the Spring release of OpenRA.  This is a huge step forward for the Dune 2000 mod in particular, but also, behind the scenes, for the still unreleased, work-in-progress Tiberian Sun mod which has received support for some crucial features during this cycle.
```
There are two spaces here after the first sentence.

penev92 commented on content/news/release-20150412.md:
```diff
@@ -0,0 +1,59 @@
+
+For modders and mappers, there have been quite a few important changes as well. We have introduced a new map format, to which maps running on this release will be updated automatically. All of the shared engine code has been moved into the OpenRA.Mods.Common project, so that third-party mods can drop their dependencies on our game mods. A lot of code has been changed to enable features that will  ...
+
+Looking forward, we hope to get the next release out faster than this time around (famous last words!). Feature-wise, you can probably expect the first appearance of the in-game map editor in the next release. Beyond that, we will see... If you want to keep track of the development, keep an eye on the [release milestone](https://github.com/OpenRA/OpenRA/milestones/Next%20release) and the [develop ...
```
"faster than this time around" is missing something

obrakmann replied:
Sounds ok to me. What do you think is missing?

penev92 commented:
I know this is about 2 weeks late, but since we missed to mention it with the correct playtest - http://www.openra.net/news/playtest-20150329/  can we mention the Dune 2000 map import now?
I still have some hopes that this will be an important milestone in attracting Dune 2000 fans to OpenRA, along with carryalls, sandworms, and (hopefully) soon more of the original game's signature features.

obrakmann commented:
Updated.

* Updated number of commits and length of cycle.
* Screenshots that need them now have cursors.
* Mentioned the d2k map import.
* Fixed the other style nits.

pchote commented on content/gamenews:
```diff
@@ -1,4 +1,4 @@
+	Title: New OpenRA release out now! This is not a drill!
 	Author: Oliver Brakmann
-	Content: Our plan to release the stable version this weekend has unfortunately been thwarted by a few show-stopper bugs. Those have now all been fixed, so a new playtest is available!\n\nIt includes, among others, fixes for the freezes that would sometimes occur mid-game, and also for crashes that happened when joining servers or selecting specific maps in the lobby. Please see the complete chan ...
+	Content: We are thrilled to announce the Spring release of OpenRA!\n\nThis release, almost six months in the making, is one of our biggest yet, with countless new features, bug fixes and improvements. Sandworms and Carryalls for Dune 2000 and playable Countries in Red Alert are the most visible enhancements, but there is lots more to discover!\n\nCheck out http://openra.net for the full announce ...
```
"This release, almost six months in the making, is one of our biggest yet, with countless new features, bug fixes and improvements." Lots of commas here.  Maybe break it up with a colon? "This release, almost six months in the making, is one of our biggest yet: it includes countless new features, bug fixes and improvements."

pchote commented on content/news/release-20150425.md:
```diff
@@ -0,0 +1,61 @@
+
+The server browser now lists games from all available mods, and also allows joining games across mods.
+
+![Mission difficulty](/images/news/20150425-cnc-mission-difficulty.png)
```
Missing cursor alert!

pchote commented on content/news/release-20150425.md:
```diff
@@ -0,0 +1,61 @@
+
+Some missions have received support for multiple difficulty levels.
+
+![Refund tooltip](/images/news/20150425-d2k-refund-tooltip.png)
```
Missing cursor alert!

obrakmann replied:
No, look closely. It's there.

pchote commented on content/news/release-20150425.md:
```diff
@@ -0,0 +1,61 @@
+We are very pleased to announce the Spring release of OpenRA.  This is a huge step forward for the Dune 2000 mod in particular, but also, behind the scenes, for the still unreleased, work-in-progress Tiberian Sun mod which has received support for some crucial features during this cycle.
```
Very long sentence here.  Could you break it up?

pchote replied:
Actually, this is repeated below, so I suggest removing it, and then combining the first sentence with the second paragraph.

pchote commented on content/news/release-20150425.md:
```diff
@@ -0,0 +1,61 @@
+Of course, we have not only improved the mods themselves. Here is a list of changes that affect the game in general:
+
+* The server browser shows and allows joining games from any installed mod.
+* It is now possible to disable the "short game" mode that has been the de-facto default in the past.
```
Technically it was the de jure default (since it was forced by the game code).  How about something like "that was forced in earlier versions"?

pchote commented on content/news/release-20150425.md:
```diff
@@ -0,0 +1,61 @@
+![Refund tooltip](/images/news/20150425-d2k-refund-tooltip.png)
+
+In sell mode, the tooltip will now show a structure's expected refund value.
+
```
It might be worth adding a TS screenshot and moving the "behind the scenes, for the still unreleased, work-in-progress Tiberian Sun mod which has received support for some crucial features during this cycle." from the first paragraph to the caption

pchote commented on content/news/release-20150425.md:
```diff
@@ -0,0 +1,61 @@
+
+<div style="text-align:center" markdown="1">
+
+![Sandworms and Carryalls](/images/news/20150118-d2k-sandworm-carryall.png)
```
I'm not expecting anyone to put in the work to do this, but it would be really awesome if we had an animated gif that showed a carryall picking up a harvester and fly off while a worm bursts out to eat another one.

penev92 replied:
Here is my attempt: http://i.imgur.com/hLj1pzZ.gif

obrakmann replied:
[How about this?](https://youtu.be/oK-paA7kRP0)

pchote replied:
:+1:

obrakmann commented:
Updated again. The TS image is probably not the best. Consider it a placeholder until someone submits something nicer ;)

pchote commented on content/news/release-20150424.md:
```diff
@@ -0,0 +1,64 @@
+* It is now possible to disable the "short game" mode that was forced in the past.
+* The player name and colour can now be set up in the settings dialog.
+* When creating a server, the game informs the host about port forwarding issues.
+* Left-click mouse orders have been markedly improved in this release.
```
"in this release" is a bit redundant.

pchote commented on content/news/release-20150424.md:
```diff
@@ -0,0 +1,64 @@
+* When creating a server, the game informs the host about port forwarding issues.
+* Left-click mouse orders have been markedly improved in this release.
+* Some missions now support difficulty levels.
+* A huge number of smaller gameplay fixes and improvements have been added. A few examples:
```
I would personally tighten this to "A huge number of smaller gameplay fixes and improvements, including:".  Not a big deal if you can't be bothered changing it, though.

pchote commented on content/news/release-20150424.md:
```diff
@@ -0,0 +1,64 @@
+* A huge number of smaller gameplay fixes and improvements have been added. A few examples:
+   * Friendly units blocking the exit of a production building will now move out of the way if needed.
+   * Units will no longer hunt cloaked units.
+   * The spy plane in Red Alert will no longer lure your anti-air capable units across the map.
```
Maybe also mention that planes don't push helicopters around anymore?

pchote commented on content/news/release-20150424.md:
```diff
@@ -0,0 +1,64 @@
+   * The spy plane in Red Alert will no longer lure your anti-air capable units across the map.
+   * Your faction will not be revealed to enemies when you have selected a random faction in the lobby.
+   * In sell mode, the tooltip will now show the expected refund value for structures.
+   * Harvesters should now no longer block themselves at refineries and also make use of all available refineries, if the nearest ones are occupied.
```
Maybe move the comma from after the second "refineries" to after the first?

pchote commented on content/news/release-20150424.md:
```diff
@@ -0,0 +1,64 @@
+* A similarly massive number of performance improvements have been made in this release.
+* Many race conditions and crashes have been fixed across the board.
+
+For the complete list of changes, visit [our changelog page](http://changelog.openra.net).
```
Maybe "please visit"?

pchote commented on content/news/release-20150424.md:
```diff
@@ -0,0 +1,64 @@
+
+![Tiberian Sun teaser](images/news/20150424-ts-teaser.png)
+
+Behind the scenes, the still unreleased, work-in-progress Tiberian Sun<br/>mod has received support for some crucial features.
```
This wording feels a bit weird.  How about `Behind the scenes, we have made significant progress towards our work-in-progress Tiberian Sun mod.<br /> Keep an eye out for more news and media about TS on our Facebook page!`?

pchote commented:
The news post looks great to me :+1:  I've left a couple more comments, but most of them are "I would have done this different" rather than actual nits.

obrakmann commented:
Updated. Fixed wording as suggested, added release id.
